### ISSUE TITLE
Really nice work, here what you can improve

### ISSUE THREAD

**Issue by Cyber4sPopo** [DorKachlon/Pokedex#1]
- [ ] You can create new pull request from the branch to the master so to we could do code review inside your pull request.
- [ ] could use one http call with fetch / xhr (it was a bonus)
- [ ] Could

**Comment by DorKachlon** [comment]
I accidentally worked and developed the project on the master, and only after everything was ready did I open a new brunch.
still relevant to do pull request?
I'm still not that savvy about GitHub :(
